<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cfwGameLauncher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="home_btn">
        <property name="text">
         <string>主页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>添加游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scan_game_btn">
        <property name="text">
         <string>批量扫描游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>删除游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setting_btn">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="about_btn">
        <property name="text">
         <string>关于</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,5">
      <item row="0" column="0">
       <widget class="QListWidget" name="gamelist_widget">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 128);
border-color: rgba(255, 255, 255, 128);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="6" rowspan="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" colspan="7">
          <widget class="QLabel" name="gamename_label">
           <property name="font">
            <font>
             <pointsize>36</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
           </property>
           <property name="text">
            <string>game_name</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="5" rowspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0" colspan="6">
          <widget class="QTextBrowser" name="metadata_browser">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
           </property>
          </widget>
         </item>
         <item row="7" column="6">
          <widget class="QTextBrowser" name="txt_browser">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" rowspan="2">
          <widget class="QPushButton" name="playgame_btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>64</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
           </property>
           <property name="text">
            <string>开始游戏</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>游戏时间</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="total_time_label">
           <property name="text">
            <string>0分钟</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="last_run_label">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
           </property>
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
           </property>
           <property name="text">
            <string>最后运行日期</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
           </property>
           <property name="text">
            <string>云状态</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
